secondIife.js

```diff
@@ -0,0 +1,37 @@
+var CarLot = (function (NewCarLot) {
+		// grab all inputs globally 
+		var carHolder = document.getElementsByClassName("carCard");
+		var newInputField = document.getElementById("inputField");
+		var descriptionInput = document.getElementById("Description");
+		var targetToEdit = "";
+		// itterates through the car cards 
+	NewCarLot.activateEvents = function() {
+	    for (var i = 0; i < carHolder.length; i++) {
+	      var containerEl = carHolder[i];
+	      // add event listener to car cards
+	      containerEl.addEventListener("click", function(event) {
+	            targetToEdit = this.childNodes[4];
```

Comment:
Using an index of a specific child node can be pretty fragile.  Imagine if someone wanted to to add another child somewhere in the carHolder element.  All of your code would now point to the wrong element.  can you think of other ways to target the element that are not as fragile?